src/index.html

```diff
@@ -12,7 +12,42 @@
       href="styles/index.scss"
     />
   </head>
+
   <body>
-    <h1>Calendar</h1>
+    <div class="main-box">
+      <div class="calendar calendar--start-day-sun calendar--month-length-31">
+        <div class="calendar__day"></div>
+        <div class="calendar__day"></div>
+        <div class="calendar__day"></div>
```

Comment:
Try to use ul and li in this case.
it's preferred because we have one repeated element inside div.
In this case best variant uses list)

Reply:
Hi. I replace all div to ul li and add new scss file for values. But Kyrylo said what we can use div, because it makes no difference :)